### DISCUSSION TITLE
Cannot drop any model. Error: Did not load native library because not agree to security policy

### DISCUSSION THREAD

**Discussion by rigeltap** [mika-f/blender-drag-and-drop#91] [General]
Help me, I got this error and cannot drop any model.

`Read prefs: "C:\Users\Axioo Pongo\AppData\Roaming\Blender Foundation\Blender\4.0\config\userpref.blend"
Info: Deleted 1 object(s)

[DRAG-AND-DROP-SUPPORT] did not load native library because not agree to security policy
drop file C:\RRR\Blender\Dimension Asset\Untitled.obj`

**Comment by mika-f** [comment]
It seems that the related documentation is out of date.
Search for `Drag and Drop Support` in the Add-ons tab of the Blender preferences windows.
After that, there will probably be a warning notice, so please check it if there is no problem.

**Reply by rigeltap** [reply]
SOLVED

After checked this option, I can drop any model.

Thank You so much 
![Blender Drang n Drop](https://github.com/mika-f/blender-drag-and-drop/assets/63064082/0e139812-ffb9-48bd-8f12-86a7df89b5da)
